tests/acceptance/new-test.js

```diff
@@ -29,14 +31,14 @@ describe('Acceptance: ember new', function() {
   beforeEach(co.wrap(function *() {
     yield tmp.setup(tmpDir);
     process.chdir(tmpDir);
-    ORIGINAL_PROCESS_ENV_CI = process.env.CI;
+    ORIGINAL_PROCESS_ENV_CI = ci.isCI;
   }));
 
   afterEach(function() {
     if (ORIGINAL_PROCESS_ENV_CI === undefined) {
-      delete process.env.CI;
+      delete ci.isCI;
```

Comment:
I don’t think we ever want to delete here. I’m also unsure how these tests can reliably pass (because now we don’t consistently control the CI state).